src/redux/actions/projectActions.ts

```diff
@@ -151,11 +151,11 @@ export function deleteProject(project: IProject)
             .find((securityToken) => securityToken.name === project.securityToken);
 
         if (!projectToken) {
-            throw new AppError(ErrorCode.SecurityTokenNotFound, "Security Token Not Found");
+            dispatch(deleteProjectAction(project));
+            throw new AppError(ErrorCode.SecurityTokenNotFound, "Security Token Not Found, only remove project from FoTT tool.");
```

Comment:
I think something like "Security token not found. Project projectName has been removed from FOTT." is more clear